src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/CsvDataExtractionService.kt

```diff
@@ -30,8 +30,11 @@ class CsvDataExtractionService(
   }
 
   private fun writeCourtBookingsToCsv(bookings: Stream<VideoBookingEvent>, csvOutputStream: OutputStream) {
+    val locationsByPrisonCode = mutableMapOf<String, List<Location>>()
```

Comment:
Revert a change here.  I started looking at cache work and the previous change was checked in by mistake.